.github/workflows/tests.yml

```diff
@@ -0,0 +1,86 @@
+---
+# Runs Tests on Pushes to `master` and Pull Requests
+name: Tests
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '**.c'
+      - '**.h'
+  pull_request:
+    paths:
+      - '**.c'
+      - '**.h'
+jobs:
+  unit-tests-legacy:
+    name: Unit Tests (legacy)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Prepare environment
+        run: |
+          ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata-all
+          sudo apt-get install -y libjson-c-dev libipmimonitoring-dev libcups2-dev libsnappy-dev \
+                                  libprotobuf-dev libprotoc-dev libssl-dev protobuf-compiler \
+                                  libnetfilter-acct-dev
```

Comment:
If we're missing these required packages in the `netdata-all` target of the `install-required-packages.sh` script, that's a bug in the script and should be handled as such.

Reply:
I have the same [comment](https://github.com/netdata/netdata/pull/8388#discussion_r391891773) here as well. Please ref an issue for this and let's move forward :)